src/components/Login/SignUp.js

```diff
@@ -25,40 +25,25 @@ class SignUp extends Component {
     };
   }
 
-  onSignUpCallback(result, error) {
-    if (error) {
-      alert(JSON.stringify(error));
-    } else if (result) {
-      if (result === undefined) {
-        Actions.login();
-      } else {
-        AsyncStorage.setItem('token', result.access_token, () => Actions.generalInfo());
-      }
-    }
-  }
-
   render() {
-    let onChangeEmail = (text) => { this.state.email = text; };
-
-    let onChangePassword1 = (text) => { this.state.password1 = text; };
-
-    let onChangePassword2 = (text) => { this.state.password2 = text; };
-
-    let signInWithFacebook = () => UserUtil.signInWithFacebook(this.onSignUpCallback.bind(this));
-
+    let onChangeEmail = (text) => this.state.email = text;
+    let onChangePassword1 = (text) => this.state.password1 = text;
+    let onChangePassword2 = (text) => this.state.password2 = text;
```

Comment:
why don't we change 
onChangePassword1 -> InputEmailAddr
onChangePassword2 -> InputSecretCode